packages/integration-tests/api/api.js

```diff
@@ -121,12 +118,13 @@ async function getAsyncOperation({ prefix, id }) {
 async function postBulkDelete({ prefix, granuleIds }) {
   const response = await callCumulusApi({
     prefix: prefix,
-    functionName: 'ApiBulkDeleteDefault',
     payload: {
       httpMethod: 'POST',
-      resource: '/bulkDelete',
-      path: '/bulkDelete',
-      pathParameters: {},
+      resource: '/{proxy+}',
+      path: '/bulkDelete/',
```

Comment:
Why did this and `providers` end their paths with a trailing slash?

Reply:
either way works